contentcuration/contentcuration/frontend/channelEdit/router.js

```diff
@@ -0,0 +1,50 @@
+import VueRouter from 'vue-router';
+import { RouterNames } from './constants';
+import TreeView from './views/TreeView';
+import store from './store';
+import EditModal from 'edit_channel/uploader/views/EditModal';
+
+const router = new VueRouter({
+  routes: [
+    {
+      name: RouterNames.TREE_ROOT_VIEW,
+      path: '/',
+      beforeEnter: (to, from, next) => {
+        return store.dispatch('channel/loadChannel', store.state.currentChannel.currentChannelId).then(channel => {
+          const nodeId = channel.main_tree;
+          return next({
+            name: RouterNames.TREE_VIEW,
+            params: {
+              nodeId,
+            },
+          });
+        });
+      },
+    },
+    {
+      name: RouterNames.TREE_VIEW,
+      path: '/:nodeId',
```

Comment:
You can navigate to a resource - as the 'preview' functionality is now outside of the edit modal in the designs here, it means that there is something meaningful about having selected a specific resource, so I think this would actually be node id - we would just need to add in some conditional behaviour in the tree view for this.